Google-SPS-Team-5/group-project
# Add new business

Changes:
- add new business form at /add.html
- write servlet to handle form and fetch businesses (endpoint is /business-data)
- implement blobstore to handle image uploads
- create constants class
- rename contactUrl in Business class to contactInfo because the method of contacting these businesses vary

To test:
- Go to endpoint /add.html (I have not added any link to it on the front end)
- Fill in a new business
- See entity at admin page or /business-data

Outstanding work:
- restrict the form to only admins, redirect general back to homepage if they chance upon the endpoint
